qa/smoke-test-multinode/src/yamlRestTest/resources/rest-api-spec/test/smoke_test_multinode/30_desired_balance.yml

```diff
@@ -186,3 +186,18 @@ setup:
   - exists: 'cluster_balance_stats.tiers.data_content.undesired_shard_allocation_count.max'
   - exists: 'cluster_balance_stats.tiers.data_content.undesired_shard_allocation_count.average'
   - exists: 'cluster_balance_stats.tiers.data_content.undesired_shard_allocation_count.std_dev'
+
+---
+"Test unassigned_shards, total_allocations, undesired_allocations and undesired_allocations_fraction":
+
+  - skip:
+      version: " - 8.11.99"
+      reason: "undesired_shard_allocation_count added in in 8.12.0"
+
+  - do:
+      _internal.get_desired_balance: { }
+
+  - gte: { 'stats.unassigned_shards' : 0 }
```

Comment:
Maybe you can merge this test with the above? Same question for 10_basic.yml file.

Reply:
Are this test cases expensive? I would prefer to keep them separate as they are added with different prs

Reply:
Sounds good. Was a nit.